modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/incoming/IncomingSnapshotCopier.java

```diff
@@ -332,7 +354,7 @@ private CompletableFuture<?> loadSnapshotMvData(ClusterNode snapshotSender, Exec
     /**
      * Requests and stores data into {@link TxStateStorage}.
      */
-    private CompletableFuture<?> loadSnapshotTxData(ClusterNode snapshotSender, Executor executor) {
+    private CompletableFuture<Void> loadSnapshotTxData(ClusterNode snapshotSender, Executor executor) {
```

Comment:
When a method is declared as returning a `CompletableFuture<?>`, you can actually return a future that completes with an int, or a String, or whatever, so it's less restricting type-wise. But when you declare it as returning a `CompletableFuture<Void>`, you can only return a future completed with a `null`, so it's stricter. I think it's better to be more strict: 'null-only' versus 'I don't care with  what this will be completed'.